lib/appsignal/probes/gvl.rb

```diff
@@ -34,8 +34,10 @@ def call
       def probe_global_timer
         monotonic_time_ns = @gvl_tools::GlobalTimer.monotonic_time
         gauge_delta :gvl_global_timer, monotonic_time_ns do |time_delta_ns|
-          time_delta_ms = time_delta_ns / 1_000_000
-          set_gauge_with_hostname("gvl_global_timer", time_delta_ms)
+          if time_delta_ns >= 0
```

Comment:
If disabled, the delta would be 0 when the measured value doesn't change, right? Do we want to keep reporting 0 or nothing at all? I think the latter.

Reply:
I'd prefer the latter as well. But even if the global timer instrumentation is disabled _at the time of measurement_, we don't know if it's been disabled _for the whole minute_, so we don't want to discard a value just because it's disabled.

Although technically zero _is_ a possible value for the delta, it is quite unlikely in any application that actually uses threads, so I'm thinking that we could just ignore zeros to achieve the same effect:

```suggestion
          if time_delta_ns > 0
```

Would that make sense for you?